app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -258,8 +258,8 @@ class PatientRepository @Inject constructor(
   fun saveOngoingEntryAsPatient(
       loggedInUser: User,
       facility: Facility,
-      supplyUuidForBpPassport: () -> UUID = { UUID.randomUUID() },
-      supplyUuidForAlternativeId: () -> UUID = { UUID.randomUUID() }
+      supplyUuidForBpPassport: () -> UUID,
```

Comment:
Here (and in other places), these supplier functions have been written because the child records (BP Passports, phone numbers, etc) may optionally be created based on the situation. We don't create it in every record in the test.

This is why I made these functions so that they will get invoked only when we are actually creating the records.